### DISCUSSION TITLE
PorousFlow - GW eq

### DISCUSSION THREAD

**Discussion by MatiasAllay** [idaholab/moose#18227] [Q&A Modules: General] [answered]
Dear MOOSE,
In think that the equations in the [GW](https://mooseframework.inl.gov/modules/porous_flow/groundwater_models.html#moose-equation-7afc6b6f-20de-4c39-b12f-2ba174c1fab2) documentation miss the source/sink term. Obviously this is well implemented since a pumping test is simulated just bellow. So where are the sources/sink implemented?
Hope that my question makes sense.
Thanks again!!
Matias

**Comment by WilkAndy** [comment, marked as answer]
There are so many different ways for implementing sources and sinks, depending on the particular physical situation.  For a start, look at https://mooseframework.inl.gov/modules/porous_flow/sinks.html and https://mooseframework.inl.gov/modules/porous_flow/boundaries.html

**Reply by MatiasAllay** [reply]
Hi Andy,
Thanks, I just wanted to know where are the sources added in the equations.
Another related question, the presentation 2 and 3 represent a HM coupled system?

**Reply by WilkAndy** [reply]
Presentation 2 can be considered a coupled HM system, but note there's no physics prescribed to "espilon" - it's just assumed to appear from somewhere.  For a true HM system you'd have to add some TensorMechanics Kernels to calculate stresses, which would give the strain epsilon.
Presentation >=3 can kind-of be considered a coupled HM system.  "epsilon" is implicit inside the Biot Modulus or specific storage, and its only physics is associated with the very simple evolution captured by Eqn(7).  A solid mechanics person would scoff at this, but it's better than nothing, since it describes the impact of changing porepressure on the porosity of the rock matrix.

**Reply by MatiasAllay** [reply]
Ahhh that makes it pretty clear. Thanks for the great documentation, I also really like the figures.
Matias
